articles/tutorials/building_2d_games/04_working_with_textures/index.md

```diff
@@ -0,0 +1,344 @@
+---
+title: "Chapter 04: Working with Textures"
+description: In this chapter, we'll look at the process of loading image files as textures using the MonoGame content pipeline and rendering them using the SpriteBatch.
+---
+
+Textures are images that are used in your game to represent the visual graphics to the player.  Before being able to render them with the [SpriteBatch](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch), you first need to create or load the texture.  We'll discuss creating a texture in code later in this series. For now, we're going to look at loading existing image files as textures and then rendering them with the [SpriteBatch](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch).
+
+## Loading a Texture
+
+Loading textures, like any other asset in MonoGame, can be done by loading the asset file directly at runtime in game, or it can be loaded through the *content pipeline*.  Both of these methods are two sides of the same coin and there are trade offs to each approach.
+
+For instance, to load an image file directly at runtime, you would need to
+
+1. Add the image file to your project
+2. Configure the project to copy the image file to the game project build directory
+3. Load the image file as a texture at runtime using the [Texture2D.FromFile](xref:Microsoft.Xna.Framework.Graphics.Texture2D.FromFile(Microsoft.Xna.Framework.Graphics.GraphicsDevice,System.String)) method.

```

Comment:
Also make the Text "Bold" for the link, as it kinda falls into the page at the moment, where you want/need it to "Pop"

Reply:
All xref links in this document and chapter 03 have been updated to be bold